client/reducers/users.js

```diff
@@ -0,0 +1,19 @@
+import { USER_CONFIGS } from '../actions/types';
+
+// TODO: Change this into a object key'd by user id?
+const initialState = {
+    users: []
+};
+
+export default (state = initialState, action) => {
+    switch (action.type) {
+        case USER_CONFIGS: {
+            const userConfigs = action.payload;
+            return Object.assign({}, state, {
+                users: userConfigs
```

Comment:
Non-blocking. I'm wondering if we should change this to to userConfigs

Reply:
ok I think I should def change the back end of "getUserConfigs" to "getUsers"